.github/workflows/run-tests.yml

```diff
@@ -0,0 +1,212 @@
+name: Run automated system tests
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+      - support/**
+    types: [opened, synchronize]
+    paths-ignore:
+      - '**/*.md'
+    release:
+      types:
+        - published
+    workflow_dispatch:
+      inputs:
+        neofs_node_ref:
+          description: 'Neofs-node ref. Examples: v0.36.0, 8fdcc6d7e798e6511be8806b81894622e72d7fdc, branch_name]'
+          required: false
+          default: ''
+
+permissions: write-all
+
+jobs:
+  run_system_tests:
+    runs-on: self-hosted
+    timeout-minutes: 120
+    steps:
+      - name: Checkout neofs-testcases repository
+        uses: actions/checkout@v3
+        with:
+          repository: nspcc-dev/neofs-testcases
+          ref: 'master'
+          path: neofs-testcases
+
+      - name: Checkout neofs-dev-env repository
+        uses: actions/checkout@v3
+        with:
+          repository: nspcc-dev/neofs-dev-env
+          ref: 'master'
+          path: neofs-dev-env
+
+      - name: Checkout neofs-node repository
+        uses: actions/checkout@v3
+        with:
+          repository: ${{ github.repository }}
+          ref: ${{ github.event.inputs.neofs_node_ref || github.head_ref || github.ref_name }}
+          path: neofs-node
+
+################################################################
+      - name: Set up Go
+        uses: actions/setup-go@v4
+        with:
+          go-version: 1.19
+      - run: go version
+
+################################################################
+      - name: Build neofs-node (neofs-cli and neofs-adm)
+        run: |
+          make all
+          echo "$(pwd)/bin" >> $GITHUB_PATH
+        working-directory: neofs-node
+
+################################################################
+      - name: Get TAG for docker images
+        run: |
+          echo "CURRENT_TAG=$( (git describe --tags --dirty --match "v*" --always --abbrev=8 2>/dev/null || cat VERSION 2>/dev/null || echo "develop") | sed 's/^v//')" >> $GITHUB_ENV
```

Comment:
`make version`?

Reply:
> `make version`?

I used the value of the VERSION variable from the Makefile:
https://github.com/nspcc-dev/neofs-node/blob/master/Makefile#L5

This is the value that appears in the TAG of the docker image:

```
$ docker images
REPOSITORY               TAG                                IMAGE ID       CREATED         SIZE
nspccdev/neofs-ir        44921688                           f222ebe2d96a   16 hours ago    43.9MB
nspccdev/neofs-storage   44921688                           f5c82ccb5be9   16 hours ago    45.5MB
```

Reply:
Fixed:
https://github.com/vvarg229/neofs-node/actions/runs/4817238441/jobs/8577703218

```
$ docker images
REPOSITORY               TAG                                IMAGE ID       CREATED              SIZE
nspccdev/neofs-ir        08b138e7                           4b277b9505c2   About a minute ago   43.9MB
nspccdev/neofs-storage   08b138e7                           f36aba6c5c3e   4 minutes ago        45.5MB
```

```
$ docker ps | grep storage
455c625cb2e9   nspccdev/neofs-storage:08b138e7   "neofs-node"             2 minutes ago   Up 2 minutes (healthy)                                  s02
416b59b5389b   nspccdev/neofs-storage:08b138e7   "neofs-node"             2 minutes ago   Up 2 minutes (healthy)                                  s03
17455d4852df   nspccdev/neofs-storage:08b138e7   "neofs-node"             2 minutes ago   Up 2 minutes (healthy)                                  s01
e8ff289e44ac   nspccdev/neofs-storage:08b138e7   "neofs-node"             2 minutes ago   Up 2 minutes (healthy)                                  s04
```

```
$ docker ps | grep ir
cd97e9b2d4eb   nspccdev/neofs-ir:08b138e7        "neofs-ir"               3 minutes ago   Up 3 minutes (healthy)                                  ir01
```